mindjunkies/forums/views.py

```
@@ -32,6 +33,9 @@ class ForumHomeView(LoginRequiredMixin, CourseContextMixin, TemplateView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
+
+        course = context["course"]
```

Ruff (0.8.2)

Local variable `course` is assigned to but never used

Remove assignment to unused variable `course`

(F841)